packages/react-conformance/src/isConformant.ts

```diff
@@ -1,15 +1,37 @@
-import * as React from 'react';
-import { IsConformantOptions, ConformanceTest } from './types';
+import { TestObject, TestingOptions } from './types';
+import { withCustomConfig } from 'react-docgen-typescript';
+import * as path from 'path';
+import { defaultTests } from './defaultTests';
 
-export function isConformant(
-  Component: React.ComponentType,
-  options: IsConformantOptions,
-  extraTests?: ConformanceTest[],
-) {
-  // const { constructorName, exportedAtTopLevel = true, packageRoot } = options;
-  // if (extraTests) {
-  //   extraTests.forEach((test: ConformanceTest) => {
-  //     test();
-  //   });
-  // }
+export function isConformant(testInfo: TestingOptions, extraTests?: TestObject) {
+  const { componentPath, displayName } = testInfo;
+  const tsconfigPath = path.join(process.cwd(), 'tsconfig.json');
+  const parser = withCustomConfig(tsconfigPath, {});
+  const components = parser.parse(componentPath);
+  const mainComponents = components.filter(comp => comp.displayName === displayName);
+
+  if (mainComponents.length === 1) {
+    const componentInfo = mainComponents[0];
+
+    for (const test of Object.keys(defaultTests)) {
+      defaultTests[test](componentInfo, testInfo);
+    }
+    if (extraTests) {
+      for (const test of Object.keys(extraTests)) {
+        extraTests[test](componentInfo, testInfo);
+      }
+    }
+  } else {
+    if (components.length === 0) {
+      throw Error('No exported components in path: ' + componentPath);
+    } else {
+      throw Error(
```

Comment:
When creating errors we generally use `new Error`, though at least [according to this](https://stackoverflow.com/questions/13294658/throw-errormsg-vs-throw-new-errormsg) they're equivalent.
```suggestion
      throw new Error('No exported components in path: ' + componentPath);
    } else {
      throw new Error(
```